### ISSUE TITLE
Wrong PSNR on the Iphone dataset in the paper Table 4

### ISSUE THREAD

**Issue by KelestZ** [facebookresearch/robust-dynrf#21]
Thanks for the great work. I run the evaluation code on iPhone dataset and noticed a wrong PSNR in Table 4. 
In the ``Block`` video (column 3), the PSNR score should be **18.04678**, rather than **18.73**.
Correspondingly, the ``average`` PSNR (last column) should  be **17.0032**, rather than **17.09**.
The other numerical scores on the iPhone dataset I got are the same as reported in Table 4.

**Comment by Li-jingfeng** [comment]
> Thanks for the great work. I run the evaluation code on iPhone dataset and noticed a wrong PSNR in Table 4. In the `Block` video (column 3), the PSNR score should be **18.04678**, rather than **18.73**. Correspondingly, the `average` PSNR (last column) should be **17.0032**, rather than **17.09**. The other numerical scores on the iPhone dataset I got are the same as reported in Table 4.

Hi, when I test the original code using conifg/Nvidia.txt, I get PSNR score equal around 12. Did you obtain similar result with me?

**Comment by KelestZ** [comment]
> > Thanks for the great work. I run the evaluation code on iPhone dataset and noticed a wrong PSNR in Table 4. In the `Block` video (column 3), the PSNR score should be **18.04678**, rather than **18.73**. Correspondingly, the `average` PSNR (last column) should be **17.0032**, rather than **17.09**. The other numerical scores on the iPhone dataset I got are the same as reported in Table 4.
> 
> Hi, when I test the original code using conifg/Nvidia.txt, I get PSNR score equal around 12. Did you obtain similar result with me?

Nope, the PSNRs are in [13, 18], same as the score being reported in this paper. Note that I didn't run their model on my own; instead, I directly used their generated results on gdrive [link](https://drive.google.com/file/d/1GzO2AEWE2azibt3XtriOHdRFj3LFPTa6/view]).
